src/runtime_src/doc/toc/security.rst

```diff
@@ -279,8 +282,51 @@ Mailbox
 
 Mailbox is used for communication between user physical function driver, xocl and management physical
 function driver, xclmgmt. The Mailbox hardware design and xclmgmt driver mailbox handling implementation
-has the ability to throttle requests coming from xocl driver which protects it from a potential DoS
-initiated by a malicious xocl driver. :ref:`mailbox.main.rst` has details on mailbox usage.
+has the ability to throttle requests coming from xocl driver. 
+
+xclmgmt driver has twofold security protections on the h/w mailbox. From packet layer, xclmgmt monitors 
+the receiving packet rates and set a threshold. If the receiving packet rates exceeds the threshold, a
+malicious behavior, ie, DoS is considered occurring from xocl, and the whole mailbox will be disabled. Only
```

Comment:
exceeds the threshold, a malicious behavior, ie, DoS is considered occurring from xocl, and the whole mailbox will be disabled --> exceeds the threshold, the mailbox is disabled which prevents the guest from sending any more commands over mailbox.

Reply:
changed